src/util/locale/no.json

```diff
@@ -146,6 +147,7 @@
       "instance-registered": "Instans registrert",
       "instance-requested-for-retry": "Instans forespurt for nytt forsøk",
       "instance-mapped": "Instans konvertert",
+      "instance-ready-for-dispatch": "Instans klar for arkivering",
```

Comment:
mer et spm egentlig, men hva betyr klar for arkivering? at den er "ferdig" hos oss men ikke sendt enda?

Reply:
Mangler vi et steg da? f.eks "sendt til destinasjon" også blir det enten "arkivert" eller "avvist" etter det om vi får svar

Reply:
Ja, det er nok fint å ha det også. Blir i så fall en event ut fra dispatch-gateway som foreløpig kun er interessant for history-service. Det er fint for oversiktlighetens skyld. Vi kn ta det opp på morgenmøtet i morgen.

Reply:
> mer et spm egentlig, men hva betyr klar for arkivering? at den er "ferdig" hos oss men ikke sendt enda?

Ja. I sammenheng med å legge til flere statushendelser, ref andre kommentaren, kan vi gjerne se på et annet navn eller navnformat for disse.